client/hwlib/src/utils.rs

```diff
@@ -0,0 +1,74 @@
+/* Copyright 2024 Canonical Ltd.
+ * All rights reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Written by:
+ *        Nadzeya Hutsko <nadzeya.hutsko@canonical.com>
+ */
+
+use once_cell::sync::Lazy;
```

Comment:
Not sure if you can update the Rust version, but `LazyLock` is now (as of last week :tada:) part of the standard library so it may be possible to drop this dependency if still required

Reply:
is it available on focal as well?

Reply:
This depends on how Rust is installed. If installed via `rustup` (e.g. from the rustup snap) then yes, otherwise if it's using the `cargo` in apt then I think no. This feature was stabilised in Rust 1.80.0

Reply:
Ah that's a shame! Good to know, thanks